### PR TITLE
make explore page gallery selects powered by sanity

### PR DESCRIPTION
**Pull request by kaitoo1**
### Summary of Changes

Replace hardcoded article on Explore page with content from Sanity

### Demo or Before/After Pics
CleanShot 2024-04-05 at 15 37 48
CleanShot 2024-04-05 at 15 37 42

### Edge Cases
na

### Testing Steps

go to /explore

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/web/src/components/Explore/GallerySelects.tsx [hunk 1]
```diff
@@ -90,7 +102,16 @@ const ArticleLinkButton = styled.div`
   }
 `;
 
-export default function GallerySelects() {
+type Props = {
+  gallerySelectsContent: CmsTypes.ExplorePageGallerySelectsList;
+};
+
+export default function GallerySelects({ gallerySelectsContent }: Props) {
+  console.log({ gallerySelectsContent });
```
Comment (Robinnnnn) [inline apps/web/src/components/Explore/GallerySelects.tsx:110]: toss log?

**Review by Robinnnnn — APPROVED** [review]
LGTM, just looks like there's a stray log
